components/RangeSlider/RangeSlider.jsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import MaterialSlider from '@material-ui/core/Slider';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import {
+  colors,
+  spacing,
+  baseFontSize as defaultBaseFontSize,
+} from '../shared/theme';
+import { hexToRgba } from '../shared';
+import valueValidator from './valueValidator';
+
+const SliderWrapper = styled.div`
+  width: auto;
+  ${({
+    theme: {
+      spacing: { large },
+    },
+  }) => `
+      padding: 0 ${large / 2}px;
```

Comment:
because the padding needs to be the half of the large size. And is not a rule the `small` size be the half, in the default theme this occur, but is not a rule.